src/ApiCallable.php

```diff
@@ -134,6 +134,40 @@ private static function setLongRunnning($callable, $longRunningDescriptor)
         return $inner;
     }
 
+    private static function callWithoutRequest($callable, $params)
+    {
+        array_shift($params);
+        return call_user_func_array($callable, $params);
+    }
+
+    private static function createGrpcStreamingApiCall($callable, $grpcStreamingDescriptor)
+    {
+        switch ($grpcStreamingDescriptor['grpcStreamingType']) {
+            case 'ClientStreaming':
+                $apiCall = function () use ($callable, $grpcStreamingDescriptor) {
+                    $response = ApiCallable::callWithoutRequest($callable, func_get_args());
+                    return new ClientStream($response, $grpcStreamingDescriptor);
+                };
+                break;
+            case 'ServerStreaming':
+                $apiCall = function () use ($callable, $grpcStreamingDescriptor) {
+                    $response = call_user_func_array($callable, func_get_args());
+                    return new ServerStream($response, $grpcStreamingDescriptor);
+                };
+                break;
+            case 'BidiStreaming':
+                $apiCall = function () use ($callable, $grpcStreamingDescriptor) {
+                    $response = ApiCallable::callWithoutRequest($callable, func_get_args());
+                    return new BidiStream($response, $grpcStreamingDescriptor);
+                };
```

Comment:
So you have ClientStream/ServerStream/BiditStream classes, but also here you have logic outside of them that are specific to their cases. Maybe the logic could be moved inside each?

```
case 'BidiStreaming':
    return BidiStream::makeApiCall($callable, $grpcStreamingDescriptor);
```

Maybe my idea is a bad design choice because it puts the code for issue calls in multiple places - let me know what you think.